icicle/include/icicle/fields/field.h

```diff
@@ -691,12 +691,9 @@ class Field
 
   static HOST_INLINE Field rand_host()
   {
-    std::random_device rd;
-    std::mt19937_64 generator(rd());
-    std::uniform_int_distribution<unsigned> distribution;
     Field value{};
     for (unsigned i = 0; i < TLC; i++)
-      value.limbs_storage.limbs[i] = distribution(generator);
+      value.limbs_storage.limbs[i] = rand();
```

Comment:
maybe assert that limbs is 4 bytes here because rand() returns int

Reply:
The implementation of limbs_storage in field is uint32 so it is an exact fit

Reply:
Apparently, rand has a maximal value of 2^15 - 1 so another solution might be better